typed-builder-macro/src/struct_info.rs

```diff
@@ -283,13 +294,23 @@ impl<'a> StructInfo<'a> {
             .as_ref()
             .and_then(|strip_bool| strip_bool.fallback.as_ref())
             .map(|fallback| (fallback.clone(), quote!(#field_name: #field_type), quote!(#arg_expr)));
-        let strip_option_fallback = field
-            .builder_attr
-            .setter
-            .strip_option
-            .as_ref()
-            .and_then(|strip_option| strip_option.fallback.as_ref())
-            .map(|fallback| (fallback.clone(), quote!(#field_name: #field_type), quote!(#arg_expr)));
+
+        let strip_option_fallback = field.builder_attr.setter.strip_option.as_ref().and_then(|strip_option| {
+            if let Some(ref fallback) = strip_option.fallback {
+                Some((fallback.clone(), quote!(#field_name: #field_type), quote!(#arg_expr)))
+            } else if let (Some(prefix), Some(suffix)) = (&strip_option.fallback_prefix, &strip_option.fallback_suffix) {
```

Comment:
Instead of handling all these cases, how about (I didn't try to compile it, so some changes may be needed to get it to typecheck):

```rust
} else if strip_option.fallback_prefix.is_none() && strip_option.fallback_suffix.is_none() {
    None
} else {
    let prefix = strip_option.fallback_prefix.as_deref().unwrap_or_default();
    let suffix = strip_option.fallback_suffix.as_deref().unwrap_or_default();
    let fallback_name = syn::Ident::new(&format!("{}{}{}", prefix, field_name, suffix), field_name.span());
    Some((fallback_name, quote!(#field_name: #field_type), quote!(#arg_expr)))
}
```